### ISSUE TITLE
Where is the implementation for CRED from ICSE'18?

### ISSUE THREAD

**Issue by junwha0511** [SVF-tools/SVF#1037]
Hi, I'm currently trying to experiment with SABER, 
and I realized that it can not deal with UAF as mentioned in #481 
But I thought it is weird because I know that your group had been published the paper for UAF detection using SVF.

Could you let me know where to use CRED?

**Comment by junwha0511** [comment]
I also want to know the bellow repo is the implementation of CRED
https://github.com/whichbug/SVF-UAF

**Comment by yuleisui** [comment]
> I also want to know the bellow repo is the implementation of CRED https://github.com/whichbug/SVF-UAF

No, I don't think so. The CRED implementation unfortunately has not been integrated into SVF yet.

**Comment by junwha0511** [comment]
Thank you for answer. you mean CRED is not open-source yet?

**Comment by yuleisui** [comment]
Not yet. Hopefully we will have someone to port or reimplement it into SVF in the future

**Comment by junwha0511** [comment]
Got it. Thank you:)
